students/ArchWu/lesson01/assignment/inventory_management/electric_appliances_class.py

```diff
@@ -0,0 +1,23 @@
+# Electric appliances class
+"""electric Applicances module"""
+from inventory_class import Inventory
```

Comment:
Since this code will be running from assignment, you will want to modify imports to be the following:
from inventory_management.inventory_class import Inventory